tests/test_models.py

```diff
@@ -294,31 +296,62 @@ def test_str(self):
 class TestClearExpired(BaseTestModels):
     def setUp(self):
         super().setUp()
-        # Insert two tokens on database.
+        # Insert many tokens, both expired and not, and grants.
+        now = timezone.now()
+        earlier = now - timedelta(seconds=100)
+        later = now + timedelta(seconds=100)
         app = Application.objects.create(
             name="test_app",
             redirect_uris="http://localhost http://example.com http://example.org",
             user=self.user,
             client_type=Application.CLIENT_CONFIDENTIAL,
             authorization_grant_type=Application.GRANT_AUTHORIZATION_CODE,
         )
-        AccessToken.objects.create(
-            token="555",
-            expires=timezone.now(),
-            scope=2,
-            application=app,
-            user=self.user,
-            created=timezone.now(),
-            updated=timezone.now(),
+
+        expired_access_tokens = AccessToken.objects.bulk_create(
+            AccessToken(token="expired AccessToken {}".format(i), expires=earlier) for i in range(100)
         )
-        AccessToken.objects.create(
-            token="666",
-            expires=timezone.now(),
-            scope=2,
-            application=app,
-            user=self.user,
-            created=timezone.now(),
-            updated=timezone.now(),
+        current_access_tokens = AccessToken.objects.bulk_create(
+            AccessToken(token="current AccessToken {}".format(i), expires=later) for i in range(100)
+        )
+
+        RefreshToken.objects.bulk_create(
+            RefreshToken(
+                token="expired AT's refresh token {}".format(i),
+                application=app,
+                access_token=expired_access_tokens[i],
+                user=self.user,
+            )
+            for i in range(100, 2)
+        )
+        RefreshToken.objects.bulk_create(
+            RefreshToken(
+                token="current AT's refresh token {}".format(i),
+                application=app,
+                access_token=expired_access_tokens[i],
+                user=self.user,
+            )
+            for i in range(49, 100, 2)
+        )
+        Grant.objects.bulk_create(
+            Grant(
+                user=self.user,
+                code="old grant code {}".format(i),
+                application=app,
+                expires=expired_access_tokens[i],
```

Comment:
I don't think you can set the access token to this DateTimeField if I'm not mistaken.
```suggestion
                expires=expired_access_tokens[i].expires,
```